### ISSUE TITLE
Add new funder logo and description

### ISSUE THREAD

**Issue by stephwright** [mozilla/science.mozilla.org#651]
We have a new funder (Siegel Family Endowment) who will be touting MozSci and our page on a blogpost coming up Aug 8th.  Just noticed we don't have their logo in our "We are supported by" section on the main page.  Could someone please add their logo and the following description next to the Sloan and Helmsley logos/descriptions:

"Siegel Family Endowment is focused on understanding and shaping learning in our technology-driven world. SFE supports people and champions organizations who work at the intersection of technology, learning, and career-readiness."

Logo coming shortly as soon as I can figure out how to change from EPS to JPG

**Comment by stephwright** [comment]
![logo-siegel](https://user-images.githubusercontent.com/14626242/28931054-b117cc9e-7829-11e7-8f73-2b1eb7e88be7.jpg)

**Comment by stephwright** [comment]
Thanks @parimatrix !!!

**Comment by parimatrix** [comment]
Happy to help ! :)

**Comment by stephwright** [comment]
Oops... just noticed that link that went with logo is actually pointing to another funder website (Sloan instead of Siegel).  I neglected to provide the new link (http://www.siegelendowment.org/)... I also shamefacedly admit I didn't even notice they were linked until now.

Also upon going to the funder website to get the above link, I discovered the image they sent me (and I provided here) has an extra "SFE" graphic on the right hand side and the logo they use does not.  Can we update the logo to the new file I'm attaching below that does NOT have that extra bit on the right side?  (Sorry!)

![logo-siegel](https://user-images.githubusercontent.com/14626242/29086788-9740680a-7c28-11e7-8269-c5554a2fc5fb.jpg)
